apps/verify/behaviours/email-lookup-sender.js

```diff
@@ -8,28 +8,27 @@ const templateId = config.govukNotify.templateUserAuthId;
 const appPath = require('../../nrm/index').baseUrl;
 const firstStep = '/start';
 const tokenGenerator = require('../models/save-token');
+const logger = require('hof/lib/logger')({ env: config.env });
 
 const getPersonalisation = (host, token) => {
   return {
-    // pass in `&` at the end in case there is another
-    // query e.g. ?hof-cookie-check
-    'link': `http://${host + appPath + firstStep}?token=${token}&`,
+    'link': `http://${host + appPath + firstStep}?token=${token}`,
     'host': `http://${host}`,
   };
 };
```

Comment:
maybe we need something like this
```
const isProduction = config.env === 'production';

const getPersonalisation = (host, token) => ({
  'link': `${isProduction ? 'https' : 'http'}://${host}${config.appPath}/start?token=${token}`,
  'host': `${isProduction ? 'https' : 'http'}://${host}`
});
```

Reply:
👍🏾 You're right, I think ASC and ACQ switch between http and https in a similar way